docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+---
+id: k8s_guide
+title: A Kubernetes tutorial
+---
+# Getting Started
+:::warning[DISCLAIMER]
+This tutorial contains some parts that are CERN-specific. <br/>
+The tools utilised to set up and to run the cluster are chosen accordingly. Feel free to use your own.
+:::
+
+In order to efficiently manage the cluster and implement continuous delivery, the following tools are required.
+
+## Kubernetes: `kubectl`
```

Comment:
These would be a lot more readable as a bulleted list

Reply:
Done.